spp_custom_field/models/cus_partner.py

```diff
@@ -20,14 +20,17 @@ def _get_view(self, view_id=None, view_type="form", **options):
         if view_type == "form":
             doc = arch
             basic_info_page = doc.xpath("//page[@name='basic_info']")
-
+            group_view = self.env["ir.ui.view"].search([("id", "=", view_id)])
+            group_view_name = self.env.ref("g2p_registry_group.view_groups_form").name
+            is_group = False
+            if group_view.name == group_view_name:
+                is_group = True
             model_fields_id = self.env["ir.model.fields"].search(
                 [("model_id", "=", "res.partner")],
                 order="ttype, field_description",
             )
 
             if basic_info_page:
-                is_group = self._context.get("default_is_group", False)
```

Comment:
The context is not working before that's why it was changed to this.

Reply:
The context is not being passed correctly with the _get_view. Only the default context are being passed not the one we added in the action.

Reply:
@emjay0921 yes that's the issue that's why in the suggested code above I used the context of the action record instead of the context of env

```
            action_id = self.env["ir.actions.act_window"].browse(options.get("action_id"))
            if basic_info_page:
                is_group = action_id.context.get("default_is_group", False)
```